src/pages/views/PageCreate.tsx

```diff
@@ -45,29 +47,53 @@ export const PageCreate: React.FC<PageCreateProps> = ({ params }) => {
   const [updateMetadata] = useMetadataUpdate({});
   const [updatePrivateMetadata] = usePrivateMetadataUpdate({});
 
+  const [selectedPageTypeId, setSelectedPageTypeId] = React.useState<string>();
+
   const {
     loadMore: loadMorePageTypes,
     search: searchPageTypes,
     result: searchPageTypesOpts
   } = usePageTypeSearch({
     variables: DEFAULT_INITIAL_SEARCH_DATA
   });
-
   const {
     loadMore: loadMorePages,
     search: searchPages,
     result: searchPagesOpts
   } = usePageSearch({
     variables: DEFAULT_INITIAL_SEARCH_DATA
   });
-
   const {
     loadMore: loadMoreProducts,
     search: searchProducts,
     result: searchProductsOpts
   } = useProductSearch({
     variables: DEFAULT_INITIAL_SEARCH_DATA
   });
+  const [selectedAttribute, setSelectedAttribute] = useState<string>();
```

Comment:
I think it's more descriptive

```suggestion
  const [focusedAttribute, setFocusedAttribute] = useState<string>();
```

Reply:
good point